docs/projects/README.md

```diff
@@ -8,7 +8,7 @@ If you've never seen a DataCamp project then you'll quickly get the gist by watc
 <center><iframe width="560" height="315" src="https://www.youtube.com/embed/HpVDKV2KALw" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe></center>
 
 > #### info::WANT TO CREATE A PROJECT? 
-> Currently only invited creators can create Projects on DataCamp, but we're always looking for engaged professionals and academics who want to help us teach Data Science. If you want to create a Project, please contact david.venturi@datacamp.com for more info!
+> Currently only invited creators can create Projects on DataCamp, but we're always looking for engaged professionals and academics who want to help us teach Data Science. If you want to create a Project, please email projects@datacamp.com for more info!
```

Comment:
Might be worth consistently making these links clickable, as you have elsewhere. `[projects@datacamp.com](mailto:projects@datacamp.com)`